Demo/UIKit/ComponentsViewController.swift

```diff
@@ -16,6 +16,12 @@ internal final class ComponentsViewController: UIViewController, PresenterExampl
         dropIn.presenter = self
         return dropIn
     }()
+
+    private lazy var advancedFlow: AdvancedFlowExample = {
+        let advancedFlow = AdvancedFlowExample()
+        advancedFlow.presenter = self
+        return advancedFlow
+    }()
```

Comment:
BTW, we need to add this to SwiftUI project as well..

Reply:
Side note: 
We might think of making a nice abstraction over it (ex `Example Protocol`) and inject those "examples" per choice, rather than instantiating them as lazy vars.

There is no point in "laziness" since we are going to call "requestInitialData" right after creation.